templates/zerver/help/change-your-language.md

```diff
@@ -1,29 +1,28 @@
 # Change your language
 
-Zulip has been translated or partially translated into dozens of
-languages. You can see which languages Zulip supports, and help add
-support for new languages on **[Transifex](https://www.transifex.com/zulip/zulip/)**.
+{!translation-project-info.md!}
 
-If your organization has a primary language other than American
-English, an administrator should also [set the organization
-notifications language][change-org-lang].
-
-[change-org-lang]: change-the-default-language-for-your-organization
-
-### Change your language
+## Change the language of the Zulip app
```

Comment:
This title makes me think that the instructions could apply to the mobile app. So maybe adding the start-end tabs and `{tab|desktop-web}` for now would be good to prep for adding the mobile instructions?